modules/defaults.php

```diff
@@ -6,14 +6,17 @@
 
 // ----------- General Variables -----------
 
+// Theme of your Node Monitor (dark or light)
+$themeChoice = 'light';
```

Comment:
Not sure whether 'light' should be the default. I'd like to keep the current theme as default.

Reply:
I feel that the Light theme aligns much more with Nano's color scheme, specifically nano.org - hence why I thought it should be default. That is an easy change, however.